CSV.js

```diff
@@ -195,6 +196,16 @@ function getValue(item, field) {
 			}
 			value += escapeValue(notes.join(valueSeparator));
 		break;
+		case 'date':
+			if (item.date) {
+				var dateISO = ZU.strToISO(item.date);
```

Comment:
I think the result will be the same in both approaches. Either we are calling the function `ZU.strToISO` an additional time (my approach) or we have to make an additional test (your approach). I don't know if performance is here important, and it might be easier to read without the additional test.